views/flower.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="flower_form_view" model="ir.ui.view">
+            <field name="name">flower.form</field>
+            <field name="model">flower</field>
+            <field name="arch" type="xml">
+                <form string="Flower">
+                    <sheet>
+                        <div class="oe_title">
+                            <label for="name"/>
+                            <h1>
+                                <field name="name" placeholder="Name"/>
+                            </h1>
+                        </div>
+                        <group>
+                            <group>
+                                <field name="scientific_name"/>
+                                <field name="season_date_start"/>
+                                <field name="season_date_end"/>
+                            </group>
+                            <group>
+                                <field name="watering_frequency"/>
+                                <field name="watering_amount"/>
```

Comment:
Since you specified in the help attribute of the field that the value should be represented in milliliters, you can show this on the form view using the following syntax:

```
<!-- Replace the watering_amount field with the following -->
<label for="watering_amount_uom"/>
<div class="o_row">
      <span><field name="watering_amount" readonly="1" nolabel="1"/></span>
      <span><field name="watering_amount_uom" readonly="1" nolabel="1"/></span>
</div>
```

For this to work, however, you need to make sure you activate Unit of Measure management in the settings.